keyboards/handwired/onekey/rpi_pico/config.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2019
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+#define PRODUCT Onekey Raspberry Pi Pico RP2040
+#define MATRIX_COL_PINS { GP4 }
+#define MATRIX_ROW_PINS { GP5 }
+#define DEBUG_MATRIX_SCAN_RATE
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP25
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 500U
+
+/* TODO */
+/* #define BACKLIGHT_PIN         A0 */
+/* #define BACKLIGHT_PWM_DRIVER  PWMD5 */
+/* #define BACKLIGHT_PWM_CHANNEL 1 */
+
+#define RGB_DI_PIN A1
+
+/* #define ADC_PIN A0 */
```

Comment:
You used the prefix A__ for the pins here, presumably for ADC pins?
I don't see this nomencalture documented or defined in `platforms/chibios/_pin_defs.h`

Reply:
These are leftovers from and don't serve any purpose as the drivers are not ready yet. I'll remove them.